docs/quickstart.md

```diff
@@ -0,0 +1,765 @@
+In this case where we see that the addition of new URL paths lead to the boilerplate code, we should lean towards the routing approach.
+!!! Info
+    If you want a more detailed explanation of these two approaches, we have explained the considerations [here](.. /core/event_handler/api_gateway/#considerations)
+The simple code might look similar to the following code snippet.
+=== "app.py"
+
+    ```python hl_lines="10-23 26-28 34-35"
+        def hello_name(event, **kargs):
+            username = event["pathParameters"]["name"]
+            return {"statusCode": 200, "body": {"message": f"hello {username}!"}}
+
+
+        def hello(**kargs):
+            return {"statusCode": 200, "body": {"message": "hello unknown!"}}
+
+
+        class Router:
+            def __init__(self):
+                self.routes = {}
+
+            def set(self, path, method, handler):
+                self.routes[f"{path}-{method}"] = handler
+
+            def get(self, path, method):
+                try:
+                    route = self.routes[f"{path}-{method}"]
+                except:
+                    print("Cannot route request to correct method")
+                    raise NotImplemented
+                return route
+
+
+        router = Router()
+        router.set(path="/hello", method="GET", handler=hello)
+        router.set(path="/hello/{name}", method="GET", handler=hello_name)
+
+
+        def lambda_handler(event, context):
+            path = event["resource"]
+            http_method = event["httpMethod"]
+            method = router.get(path=path, method=http_method)
+            return method(event=event)
+    ```
+
+=== "template.yaml"
+
+    ```yaml hl_lines="15-25"
+        AWSTemplateFormatVersion: "2010-09-09"
+        Transform: AWS::Serverless-2016-10-31
+        Description: >
+            Sample SAM Template for powertools-quickstart
+        Globals:
+            Function:
+            Timeout: 3
+        Resources:
+            HelloWorldFunction:
+            Type: AWS::Serverless::Function
+            Properties:
+                CodeUri: hello_world/
+                Handler: app.lambda_handler
+                Runtime: python3.9
+                Events:
+                    HelloWorld:
+                        Type: Api
+                        Properties:
+                        Path: /hello
+                        Method: get
+                    HelloWorldName:
+                        Type: Api 
+                        Properties:
+                        Path: /hello/{name}
+                        Method: get
+        Outputs:
+            HelloWorldApi:
+                Description: "API Gateway endpoint URL for Prod stage for Hello World function"
+                Value: !Sub "https://${ServerlessRestApi}.execute-api.${AWS::Region}.amazonaws.com/Prod/hello/"
+    ```
+
+* We added two `hello_name` and `hello` methods (line 1-7). 
```

Comment:
either remove the word two or bring the word method to the front:
- We added two methods: `hello_name` and `hello`
- We added `hello_name` and `hello` methods.

Reply:
fixed